### PR TITLE
improve ruby lint

### PR DESCRIPTION
**Pull request by lucasmellos**

### CODE REVIEW

File: .github/actions/test-ruby/action.yml [hunk 1]
```diff
@@ -40,6 +40,15 @@ runs:
         restore-keys: |
           ${{ runner.os }}-gems-
 
+    - name: Linting
+      run: |
+        bin/rails zeitwerk:check
+        bundle exec brakeman --skip-files bundle/ -q -w2
+        gem install rubocop
```
Comment (ryanwi) [inline .github/actions/test-ruby/action.yml:47]: I need to check with Bryan and Geoff why we were installing each time and not adding to the Gemfile. Perhaps we can assume it's installed here.

Comment (lucasmellos) [reply .github/actions/test-ruby/action.yml:47]: that'd be great to have, the caching will do the job for us

Comment (ryanwi) [reply .github/actions/test-ruby/action.yml:47]: or could have a flags for each of the checks?

Comment (lucasmellos) [reply .github/actions/test-ruby/action.yml:47]: we can, for sure. Just FYI flags will be enabled by default.

Comment (lucasmellos) [reply .github/actions/test-ruby/action.yml]: changes were implemented

File: .github/actions/test-ruby/action.yml [hunk 2]
```diff
@@ -40,14 +49,30 @@ runs:
...
+
+    - name: Run brakeman
+      if: inputs.BRAKEMAN == 'true'
+      uses: devmasx/brakeman-linter-action@v1.0.0
+      env:
+        GITHUB_TOKEN: ${{github.token}}
+
+    - name: Run rubocop
+      if: inputs.RUBOCOP == 'true'
+      uses: gimenete/rubocop-action@1.0
```
Comment (ryanwi) [inline .github/actions/test-ruby/action.yml:66]: in prime, there is a git diff in combination to detect which files are changing and only run rubocop for that - https://github.com/signalwire/prime-rails/blob/master/.github/workflows/ci.yml#L30

prime is probably the exception, as we'd generally want to run it for everything.  So, prime could set this to false and run its own maybe.

**Review by ryanwi — APPROVED** [review]
let's try it

**Pull request merged**